dashboard/models.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
i don't agree with this, but i'll go with the crowd on this one. 

or what it's worth: a 'dashboard' is a _front-end_ concept. our data modeling should not have any idea what a 'dashboard' is. these are two models that relate directly to courses and course runs. if i'm looking for a final grade that a student got in a course run, i would expect to find that model in `courses/models.py`. same thing if i'm looking for a student's enrollment in a course. it just makes sense. the fact that we _show_ the grades in the dashboard on the front-end is, in my opinion, irrelevant to where we put these models. any API could ask for enrollment/certificate information. the fact that we have an api module in dashboard, and that happens to ask for enrollment/certificate information, shouldn't be a guide to where we put the models.

but again, i'll go with the crowd